src/aroma/aroma.c

```diff
@@ -158,8 +158,11 @@ byte libaroma_start() {
   
 #ifdef LIBAROMA_TTY_KDSETMODE
   ALOGI("KDSETMODE = KD_GRAPHICS");
-  int tty1 = open("/dev/tty1", O_RDWR);
+  char ttydev[32];
+  snprintf(ttydev,32,"/dev/tty%i",LIBAROMA_TTY_KDSETMODE);
+  int tty1 = open(ttydev, O_RDWR);
   ioctl(tty1, KDSETMODE, KD_GRAPHICS);
```

Comment:
I know you didn't change this line in this PR but I just noticed: you forgot to check the return value of `ioctl`.